docs/modding-and-development/development/contributing-code-to-northstar.md

```diff
@@ -30,3 +30,51 @@ To get the cropped GIFs of a screenrecording:
 3. Use a site like [https://ezgif.com/video-to-gif](https://ezgif.com/video-to-gif) for cropping and turning it into a GIF \
    (The linked site has not been vetted, use at own risk)
 4. Upload GIF to GitHub pull request description by just pasting it at the appropriate location.
+
+### Avoiding "PR hell"
+
+_"PR hell"_ is what contributors refer to when a pull request is stuck as pending without any indication for progress.
+
+There can be a variety reasons for this. Whether it's cause the pull request was overlooked, the changes are too big that anyone actually dares looking at it, or simply every reviewer is too busy at the moment, not getting a response on something you put a lot of work into is a terrible experience.
+
+While we are continuously working on improving the expierence and recruiting more reviewers, here's a few tips to avoid _"PR hell"_.
+
+**Make sure your PR is easy to review:**
+
+If you are submitting a new feature, highlight it with screenshots, screenrecordings, GIFs etc. If you can make your change look interesting, it highly increases the chance of someone becoming interested in getting it merged.
+
+If you are submitting a bug fix, make sure to mention the steps to reproduce the bug and that the same steps no longer cause the bug on your fixed version.
+
+Finally, smaller code changes are easier to review than bigger ones, which brings us to:
+
+**Keep PRs concise and focused on a specific issue:**
+
+While working on code to create some feature or fix some issue you might come across some other things you also want to change. While this is fine, it's highly recommended to split out these changes into separate PRs.
+
+Having lots of small PRs is way easier to review and work with than one big PR. Further if there's an issue with one of the changes it does not block the whole big PR but just one of the small ones.
+
+Smaller changes are also way easier to review and test.
+
+If you're working on a big feature that introduces lots of changes (let's say you implemented an entire party system into Northstar) a recommended approach is to PR your big change so that feature can be easily observed and interest generated. \
```

Comment:
```suggestion
If you're working on a big feature that introduces lots of changes (let's say you implemented an entire party system into Northstar) a recommended approach is to PR your big change so that feature can be easily observed and interest generated.
```
Trailing backslash?

Reply:
It's actually [markdown syntax to cause a newline without starting a whole new paragraph](https://www.markdownguide.org/basic-syntax#line-break-best-practices) ^^